pkg/handlers/v1/transformer_test.go

```diff
@@ -121,18 +121,18 @@ func TestTransformEC2(t *testing.T) {
 				},
 				Changes: []Change{
 					{
-						PrivateIPAddresses: []string{},
-						PublicIPAddresses:  []string{"34.219.72.29"},
-						Hostnames:          []string{"ec2-34-219-72-29.us-west-2.compute.amazonaws.com"},
-						ChangeType:         "ADDED",
+						// PrivateIPAddresses: []string{},
```

Comment:
Question: I think I missed how this happens when I was reading through. What changed that made us omit the PrivateIPAddresses array when it's empty (to be clear, I think that's an improvement!)

Reply:
Also its probably worth deleting commented out code if were not going to use it.

Reply:
The PrivateIPAddresses array is now nil if there are no items, instead of an empty array.
- [ ] Delete commented out code